src/main/kotlin/nevam/ReleaseCommand.kt

```diff
@@ -192,6 +199,9 @@ class ReleaseCommand : CliktCommand(name = "release") {
     waitTillAvailable().echoStreamingProgress()
   }
 
+  private fun NexusModule.contentUrl(repository: StagingProfileRepository, pom: Pom) =
+    "$repositoryUrl/content/repositories/${repository.id}/${pom.coordinates.mavenGroupDirectory()}"
```

Comment:
Wanna make this public to write a test for this? It'll be bad, but still less than not having a test.

Reply:
Good point; made it an `internal` extension and tested it.

Reply:
Awesome, let's merge this!